webapp/app/[locale]/tunnel/_components/reviewOperation/step.tsx

```diff
@@ -60,16 +56,23 @@ const Completed = function ({
       <div className="mt-4">
         <PositionStatus position={position} status={ProgressStatus.COMPLETED} />
       </div>
-      <div className="flex w-full flex-col">
-        <div className={`${upperBoxCommonCss} z-10 -mb-3 bg-neutral-50`}>
-          <span className="mr-auto text-neutral-500">{description}</span>
-        </div>
-        <div className={bottomBoxCommonCss}>
-          <GreenCheckIcon />
-          <span className="mr-auto text-emerald-500">{t('confirmed')}</span>
-          <SeeOnExplorer chainId={explorerChainId} txHash={txHash} />
-        </div>
-      </div>
+      <TwoRowBox
+        bottom={
+          <>
+            <GreenCheckIcon />
+            <span className="mr-auto text-emerald-500">{t('confirmed')}</span>
+            <SeeOnExplorer chainId={explorerChainId} txHash={txHash} />
+          </>
+        }
+        top={{
+          bgColor: 'bg-neutral-50',
+          children: (
+            <>
+              <span className="mr-auto text-neutral-600">{description}</span>
+            </>
+          ),
```

Comment:
I think we don't need this fragment

```suggestion
          children: (
             <span className="mr-auto text-neutral-600">{description}</span>
          ),
```

Reply:
You're correct, it is not needed indeed!

Reply:
Removed in 35cd5f2